packages/core/src/components/Dialog/Dialog.tsx

```diff
@@ -49,9 +49,12 @@ export interface HvDialogProps
   disableBackdropClick?: boolean;
   /** A Jss Object used to override or extend the styles applied to the component. */
   classes?: HvDialogClasses;
+  /** Variant of the dialog. Adds a status bar to the top of the dialog. If not provided, no status bar is added. */
+  variant?: "success" | "error" | "warning";
```

Comment:
Alignment with Banner/Snackbar is nice 👍 

But what if users want a **catastrophic** Dialog? Should we allow any semantic color? 🤔 DS also calls these _Negative_, _Warning_, and _Positive_.

To me it's fine either way. If eventually we need to support any `semantic`/`color`, we could always deprecate this in favour of a more generic color prop